bucket/oss-browser.json

```
@@ -0,0 +1,39 @@
+{
+    "version": "1.9.5",
+    "description": "View, upload, download, and manage files and folders with ease.",
+    "homepage": "https://github.com/aliyun/oss-browser",
+    "license": "Apache-2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://oss-attachment.cn-hangzhou.oss.aliyun-inc.com/oss-browser/1.9.5/oss-browser-win32-x64.zip",
+            "hash": "9b3e96edc3946317cbab30e7a6fee6ac8504d214737b2449d5a83c3c2db496a7",
+            "extract_dir": "oss-browser-win32-x64"
+        },
+        "32bit": {
+            "url": "https://oss-attachment.cn-hangzhou.oss.aliyun-inc.com/oss-browser/1.9.5/oss-browser-win32-ia32.zip",
+            "hash": "2d01ad6e8fb3fd0b8f5c338a58364dc366232b1cfd5b5924b54fd8f9cf55aafd",
+            "extract_dir": "oss-browser-win32-ia32"
+        }
+    },
+    "bin": "oss-browser.exe",
+    "shortcuts": [
+        [
+            "oss-browser.exe",
+            "oss-browser"
```

On screenshot the main window contain this title:

```suggestion
            "OSS Browser"
```